portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -23,10 +34,51 @@
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
+  DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
```

Comment:
it is a good practice to always put private by default
also, this variable seems to be fine to be set as static